python/_tskitmodule.c

```diff
@@ -7113,6 +7113,33 @@ TreeSequence_genealogical_nearest_neighbours(TreeSequence *self, PyObject *args,
     return ret;
 }
 
+/* Forward Declaration */
+static PyTypeObject TreeSequenceType;
+
+static PyObject *
+TreeSequence_get_kc_distance(TreeSequence *self, PyObject *args, PyObject *kwds)
+{
+    PyObject *ret = NULL;
+    TreeSequence *other = NULL;
+    static char *kwlist[] = {"other", "lambda_", NULL};
+    double lambda = 0;
+    double result = 0;
+    int err;
+
+    if (!PyArg_ParseTupleAndKeywords(args, kwds, "O!d", kwlist,
+                &TreeSequenceType, &other, &lambda)) {
+        goto out;
+    }
```

Comment:
There's a ``TreeSequence_check_state`` function (or something) that we usually call here to make sure that the input TreeSequence object is correctly initialised. Probably overly paranoid, but should call it here for consistency.